src/sentry/plugins/sentry_mail/activity/base.py

```diff
@@ -64,10 +64,11 @@ def get_project_link(self):
 
     def get_group_link(self):
         return absolute_uri(
-            '/{}/{}/issues/{}/'.format(
+            '/{}/{}/issues/{}/?environment={}'.format(
```

Comment:
Won't this value also need to be escaped before just blindly being shoved into a querystring? I'd rather we pass it through [urlencode](https://docs.python.org/2/library/urllib.html#urllib.urlencode).

Otherwise, there's possibility for like, an environment name with a ? in it, or even XSS/HTML injection.